### DISCUSSION TITLE
I have 7 sensors that use Bluetooth 5.2 and provide 70 readings per second. Therefore, with 7 sensors, I would have a total of 70 * 7 readings. I am considering using react-native-ble-manager. Is it worth using this package?

### DISCUSSION THREAD

**Discussion by sunny812546** [innoveit/react-native-ble-manager#1179] [Q&A]
I have 7 sensors that use Bluetooth 5.2 and provide 70 readings per second. Therefore, with 7 sensors, I would have a total of 70 * 7 readings. I am considering using react-native-ble-manager. Is it worth using this package?

**Comment by marcosinigaglia** [comment]
I think you have to use startNotificationUseBuffer or you flood the react native bridge. Or create a customized version of the library.
